packages/tour-kit/package.json

```diff
@@ -34,11 +34,11 @@
 		"@popperjs/core": "^2.10.2",
 		"classnames": "^2.3.1",
 		"react": "^17.0.2",
-		"react-popper": "^2.2.5",
-		"typescript": "^4.4.4",
-		"use-debounce": "^3.1.0"
+		"react-popper": "^2.2.5"
 	},
 	"devDependencies": {
+		"@automattic/calypso-color-schemes": "workspace:^",
```

Comment:
This is for the import [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/tour-kit/.storybook/preview.js#L1) right? It doesn't look like we need those styles right now, so we can probably remove the import instead.

Reply:
Yes, we're currently taking care of "Code style (Web app)" test issue in #58658 😃

Reply:
Looks like both PRs are complementary. Do you think we can merge this PR into #58658? Otherwise I can wait until #58658 is done and rebase this.

Reply:
We had planned to wait for this to get merged and rebase #58658, but either way works. So feel free to merge if you get to work on it, otherwise we will take care of both in afternoon (utc).